pkg/operator/generate.go

```diff
@@ -5,7 +5,7 @@ package operator
 
 // build the kubenetes client
 //go:generate go run ../../vendor/sigs.k8s.io/controller-tools/cmd/controller-gen object paths=./apis/...
-//go:generate sh -c "cd ../../../../.. && go run github.com/Azure/ARO-RP/vendor/k8s.io/code-generator/cmd/client-gen --clientset-name versioned --input-base github.com/Azure/ARO-RP/pkg/operator/apis --input aro.openshift.io/v1alpha1,preview.aro.openshift.io/v1alpha1 --output-package github.com/Azure/ARO-RP/pkg/operator/clientset --go-header-file github.com/Azure/ARO-RP/hack/licenses/boilerplate.go.txt"
+//go:generate go run ../../vendor/k8s.io/code-generator/cmd/client-gen --clientset-name versioned --input-base ./apis --input aro.openshift.io/v1alpha1,preview.aro.openshift.io/v1alpha1 --output-package ./clientset --go-header-file ../../hack/licenses/boilerplate.go.txt
```

Comment:
I _used_ to put aro-rp in ~/git/ARO-RP, but switched to my `$GOPATH`  because of this command. I agree, I do not think it should be a requirement to be in the `$GOPATH`.